TRB_label_assist/clustering.py

```diff
@@ -98,19 +119,23 @@ def add_loc_clusters(
             loc_df.loc[:, f"{loc_type}_DBSCAN_clusters_{r}_m"] = labels
 
     elif alg == 'naive':
+
+        cleaned_loc_entry= CleanEntryTypeData(loc_df,loc_entry)
+
         for r in radii:
             # this is using a modified Similarity class that bins start/end
             # points separately before creating trip-level bins
-            sim_model = eamts.Similarity(loc_df,
-                                         radius_start=r,
-                                         radius_end=r,
-                                         shouldFilter=False,
-                                         cutoff=False)
```

Comment:
I don't see these in the new model config. Are the default set to these values?

Reply:
Yes.

Reply:
where is `shouldFilter`?

Reply:
This has not yet been answered

Reply:
Added back. Was removed by mistake.